README.md

```diff
@@ -43,26 +44,48 @@ To test new features, you can use ydb version that is currently in testing mode
 or use a build from `main` brunch with `ghcr.io/ydb-platform/local-ydb:trunk` image.
 Also you can set specific version like `cr.yandex/yc/yandex-docker-local-ydb:23.1`
 
-### Custom backend in dev mode
+### Custom configuration in dev mode with .env file
 
-YDB docker represents a single node cluster with only one version, small amount of storage groups, PDisks and VDisks. It may be not enough for development purposes. If you have your own development cluster with sufficient amount of entities, you can run the app in the dev mode with this cluster as backend. To do it, set you host to `REACT_APP_BACKEND` variable in `dev` script. For example:
+You can run the app with your own params by adding `.env` file to project root. There is an example in `.env.example`.
+
+1. Add `.env` file to project root.
+
+```
+DISABLE_ESLINT_PLUGIN=true
```

Comment:
Let's remove content of .env.example file here and add bash script:
```
\```shell script
cp .env.example .env
\```
```